pkgs/sagemath-doc-html/meson.build

```diff
@@ -0,0 +1,21 @@
+project('sagemath-doc-html',
+        version: run_command('cat',
```

Comment:
ok, thanks

Reply:
Unfortunately doesn't work before `project` has been called.
```diff
diff --git a/pkgs/sagemath-doc-html/meson.build b/pkgs/sagemath-doc-html/meson.build
index 3d7763d27f4..dcdd9838813 100644
--- a/pkgs/sagemath-doc-html/meson.build
+++ b/pkgs/sagemath-doc-html/meson.build
@@ -1,7 +1,5 @@
 project('sagemath-doc-html',
-        version: run_command('cat',
-                             join_paths(meson.current_source_dir(), 'VERSION.txt'),
-                             check: true).stdout().strip()
+        version: import('fs').read(join_paths(meson.current_source_dir(), 'VERSION.txt')).strip()
 )
 
 docdir = get_option('datadir') / 'share' / 'doc' / 'sage'
```
... gives the error 
```
[sagemath_doc_html-10.2.rc3]   File "/usr/local/lib/python3.12/site-packages/mesonbuild/modules/__init__.py", line 56, in __init__
[sagemath_doc_html-10.2.rc3]     self.project_version = interpreter.build.dep_manifest[interpreter.active_projectname].version
[sagemath_doc_html-10.2.rc3]                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[sagemath_doc_html-10.2.rc3] KeyError: ''
[sagemath_doc_html-10.2.rc3] The Meson build system
[sagemath_doc_html-10.2.rc3] Version: 1.2.3
[sagemath_doc_html-10.2.rc3] Source dir: /Users/mkoeppe/s/sage/sage-rebasing/worktree-clean/pkgs/sagemath-doc-html
[sagemath_doc_html-10.2.rc3] Build dir: /Users/mkoeppe/s/sage/sage-rebasing/worktree-clean/pkgs/sagemath-doc-html/.mesonpy-ux25es98
[sagemath_doc_html-10.2.rc3] Build type: native build
[sagemath_doc_html-10.2.rc3] 
[sagemath_doc_html-10.2.rc3] ../meson.build:2:30: ERROR: Unhandled python exception
```

Reply:
I'll replace the cat call by a simple python script

Reply:
Actually, portably calling the Python interpreter can also not be done before the `project` call has finished. 
So I'll leave concerns about portability to Windows to a follow-up when any of this can actually be tested on Windows.

Reply:
its even simpler: `files('version.txt')`

Reply:
Thanks for finding this, done.